renormalizer/mps/tests/test_mp.py

```diff
@@ -59,3 +59,80 @@ def test_environ():
         e = complex(tensordot(l, r, axes=((0, 1, 2), (0, 1, 2)))).real
         assert pytest.approx(e) == mps.expectation(mpo)
 
+
+@pytest.mark.parametrize("comp", (True, False))
+@pytest.mark.parametrize("mp", ("mps", "mpdm", "mpo" ))
+def test_svd_compress(comp, mp):
```

Comment:
With this test suite, `test_svdqn.py` becomes unnecessary. We can delete `test_svdqn.py` and `test_svd_qn.npz`.

Reply:
Agree.